pkg/virt-controller/watch/migration.go

```diff
@@ -704,7 +707,7 @@ func (c *MigrationController) handleTargetPodCreation(key string, migration *vir
 	// migration was accepted into the system, now see if we
 	// should create the target pod
 	if vmi.IsRunning() {
-		if err := c.createPDB(key, vmi, migration); err != nil {
```

Comment:
After some discussion, we probably want to see if this change of behavior is desired. Adding hold for now.

Reply:
The outcome of the discussion was that this behaviour is not advertised and expected thus removable for the scope of this PR.